site/app/templates/polls/PollForm.twig

```diff
@@ -163,7 +163,7 @@
         <span id="empty-response-error" class="red-message polls-submit-error" hidden>Please make sure all responses are filled in.<br/></span>
         <span id="file-size-error" class="red-message polls-submit-error" hidden>Please make sure to upload a file that does not exceed the maximum size.<br/></span>
         <a href="{{base_url}}" class="btn btn-danger">Cancel</a>
-        <button type="submit" id="poll-form-submit" class="btn btn-primary" {{ poll is null ? 'disabled' }}>{{ poll is null ? 'Add' : 'Edit' }} Poll</button>
+        <button type="submit" id="poll-form-submit" class="btn btn-primary" {{ poll is null ? 'disabled' }}>{{ poll is null ? 'Add' : 'Save Poll' }} </button>
```

Comment:
Also revert the irrelevant structural change on how this button's text is formulated. Only change the part where it says 'Edit'.

Reply:
Ok, thank you very much for the detailed feedback! Changes are made